### PR TITLE
Added ID Modification Log to the Access Report Print

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request

Allows MPs (or others) to read who made what changes to an ID card by plugging the card into an ID Computer and hitting the print button.
This button previously printed a list of all the accesses on the ID Card, and will still do this at the same time.

# Explain why it's good for the game

It can be incredibly frustrating to figure out how someone can suddenly open a door without permission and is essentially untraceable at the moment without admin help.

# Testing Photographs and Procedure
image

# Changelog
:cl:
add: Added ID Modification Log to the Access Report printout from an ID Console.
/:cl:

### CODE REVIEW

File: code/modules/cm_marines/marines_consoles.dm [hunk 1]
```diff
@@ -91,18 +91,11 @@
...
-					if(isnull(target_id_card.faction_group))
-						target_id_card.faction_group = list()
-					else
-						faction = target_id_card.faction_group
 					var/contents = {"<center><h4>Access Report</h4></center>
 								<u>Prepared By:</u> [user_id_card?.registered_name ? user_id_card.registered_name : "Unknown"]<br>
 								<u>For:</u> [target_id_card.registered_name ? target_id_card.registered_name : "Unregistered"]<br>
 								<hr>
-								<u>Faction:</u> [faction]<br>
+								<u>Faction:</u> [target_id_card.faction ? target_id_card.faction : "N/A"]<br>
```
Comment (Drulikar) [inline code/modules/cm_marines/marines_consoles.dm:98]: These changes do not look to be correct. See your own testing example image where it says the faction is a list.

File: code/modules/cm_marines/marines_consoles.dm
Comment (Drulikar) [inline code/modules/cm_marines/marines_consoles.dm:1]: Please also update 

`/obj/structure/machinery/computer/card/ui_act(action, params)` to
`/obj/structure/machinery/computer/card/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)`

And update `var/mob/user = usr` to
`var/mob/user = ui.user`

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/modules/cm_marines/marines_consoles.dm (hunk 1 above)
Comment (realforest2001) [reply code/modules/cm_marines/marines_consoles.dm:98]: Changes are after the image, deleted stuff directly above this line fixes that for some reason it was being set to a list variable.

Comment (realforest2001) [reply code/modules/cm_marines/marines_consoles.dm:98]: Image updated

**Review by Drulikar — APPROVED** [review]

**Pull request merged**